### DISCUSSION TITLE
Termux: Integrating and harmonising ble.sh with bash_it

### DISCUSSION THREAD

**Discussion by b9Joker108** [akinomyoga/ble.sh#432] [Q&A]
Hello

In Termux, I have just recently first installed ble.sh, followed by bash-it. Bash-it, true to its name, has clobbered my ble.sh. I would really like to integrate and harmonise these two projects locally. I have not yet read the documentation for both projects. This post, is really a call for guidance, for how I may go about this from the ble.sh side, and what I should really know, and what I should really, expressly, read and take notice of, in the documentation. I am surprised, as I just sourced my backup .bashrc prior to installing bash_it, but ble.sh was not reinstituted for that shell, which confuses me.

All assistance greatly appreciated.

Cheers
Beauford

**Comment by akinomyoga** [comment]
This is a duplicate of https://github.com/Bash-it/bash-it/discussions/2250
